src/js/ripple/remote.js

```diff
@@ -373,16 +377,49 @@ Remote.prototype._trace = function() {
   }
 };
 
+
+
+// Generate KeyPair from given seed. 
+Remote.prototype.generateKeyPair = function(secret, id) {
```

Comment:
https://github.com/ripplerm/ripple-lib/commit/8c73e4e8f968e82170a402a9b2bd129d0ae3d2d7

still had to keep the seed.get_key() part, so that bin/rsign won't fail.